feat: require LXD 5.0 for the appliance (#169)
anbox-cloud/discourse-docs: howto/install-appliance/finalise.md

After installing the Anbox Cloud Appliance, you must initialise it and register your Ubuntu SSO account with the dashboard.

> <a name="initialise"></a>

Reviewer: The anchor should stay with the "Initialise" title below.

Author: Fixed

## Install requirements

> LXD >= 5.0 is required for the Anbox Cloud Appliance. Check which version you have currently installed:

Reviewer: ```suggestion
The Anbox Cloud Appliance requires LXD >=5.0. Check which version you have currently installed:
```

    lxd --version

> If LXD is not installed, run

Reviewer: ```suggestion
If LXD is not installed, run:
```

    snap install --channel=5.0/stable lxd

> If LXD is already installed but the version is older than 5.0, run

Reviewer: ```suggestion
If LXD is already installed but the version is older than 5.0, run:
```
